paddle/operators/math/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language:        Cpp
+BasedOnStyle:  Google
```

Comment:
This file is not needed, Because it is same as `operators/.clang-format`. clang-format will find the nearest .clang-format.